keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -0,0 +1,860 @@
+<!--
```

Comment:
You will need PRR approval if you want to target 1.27 - happy to take this.

Reply:
@wojtek-t thanks! Do you mean you can help review the PRR section of this doc?

Reply:
You also need to add appropriate file under:
https://github.com/kubernetes/enhancements/tree/master/keps/prod-readiness/sig-node

And yes, I queued that and will take a look later this week.

Reply:
Thanks @wojtek-t ! Added the prod-readiness yaml file.